doc/en/api/1.0.0/styles.yaml

```diff
@@ -192,12 +187,7 @@ paths:
         Using PUT with any other format will modify the content of the style. You can also use PUT with a ZIP file to upload a SLD 1.0 (`application/vnd.ogc.sld+xml`) file and any associated icon files
 
       parameters:
-        - name: styleBody
-          description: Style body information to modify.
-          in: body
-          required: true
-          schema:
-            $ref: "#/definitions/StyleInfoPost"
+        - $ref: "#/definitions/StyleInfoPost"
```

Comment:
you are giving up "to upload" vs "to modify"; I expect that is fine.

Reply:
Yeah, the description of the PUT / POST itself should describe the distinction fine. Not really needed in the description of the payload as well.